src/liblsquic/lsquic_handshake.c

```diff
@@ -2346,6 +2364,9 @@ handle_chlo_frames_data(const uint8_t *data, int len,
         break;
     }
 
+    rtt = determine_rtts(enc_session, ip, t);
```

Comment:
Moving of this code, while an improvement, does not seem related to the fix?

Reply:
I take it back! :-)

Reply:
> I take it back! :-)

You must not take it back, because enc_session->cert_ptr is set after get_sni_SSL_CTX.
If you move it back, will coredump.

Reply:
By "take back," I meant that I retract my original comment.